packages/browserslist-config-terra/src/terra-dev-site/browserslist-config-terra/UpgradeGuide.tool.mdx

```diff
@@ -0,0 +1,99 @@
+# Browserslist Config Terra Upgrade Guide
```

Comment:
Why don't we move this doc to terra-toolkit-docs?

Reply:
I had almost forgotten that project existed. Moved into terra-toolkit-docs:

https://github.com/cerner/terra-toolkit/pull/511/commits/d52f68a3b2a5bf72cffaa255f1cea72aa1b7f1e8